npc/other/CashShop_Functions.txt

```diff
@@ -263,6 +263,26 @@ function	script	F_CashSiegeTele	{
 	return;
 }
 
+// Siege Teleport Scroll 2 (WoE 1.5)
+//============================================================ 
+// - Warp player to selected guild castle.
+// - No arguments.
+function	script	F_CashSiegeTele2	{
+	switch(select("Mardoll (arug_cas01):Cyr (arug_cas02):Horn (arug_cas03):Gefn (arug_cas04):Badanis (arug_cas05):Himinn (schg_cas01):Andlangr (schg_cas02):Vidblainn (schg_cas03):Hljod (schg_cas04):Schatirnil (schg_cas05)")) {
+	case 1: warp "aru_gld",159,272; end;
```

Comment:
Not sure if we should fix it here, but the end here stops the script. The return at the bottom will never be ran thus anything in the item script after the callfunc will also never be ran.

Since this is a mimic from the other items those are "broken" as well.

Reply:
Ignore me. It's been a while since I worked on a script. I think we put ends after warps on purpose to detach the player from the script to avoid any possible issues.